.github/workflows/industrial_ci.yml

```
@@ -16,12 +16,14 @@ jobs:
   industrial_ci:
     strategy:
```

RollingのCIで失敗するとJazzyのCIが始まらなくなってしまうので、下記のPRを参考に設定の追加お願いします。
https://github.com/rt-net/rt_manipulators_cpp/pull/34/files
```suggestion
    strategy:
        fail-fast: false
```